.changeset/server-fix-fga-import-on-old-core.md

```
@@ -0,0 +1,18 @@
+---
+'@mastra/server': patch
+'@mastra/deployer': patch
+---
+
+Fix runtime import failures when `@mastra/server` is paired with an older `@mastra/core` than its own version.
+
+In 1.32.0, `@mastra/server` added top-level value imports of names that only exist in `@mastra/core@1.32.0+` (`MastraFGAPermissions` from `@mastra/core/auth/ee`; `branchesFilterSchema`, `branchesOrderBySchema`, `getBranchArgsSchema`, `listBranchesResponseSchema`, `getBranchResponseSchema` from `@mastra/core/storage`; `computeNextFireAt` from `@mastra/core/workflows`). When a user's project pinned `@mastra/core@1.31.0` but transitively pulled `@mastra/server@1.32.0` through `mastra` / `@mastra/deployer`, the bundled output crashed at module link time:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)